app/assets/stylesheets/application.css.scss

```diff
@@ -56,6 +56,11 @@
 .trophy {
   padding-top: 210px;
 }
+
+.checkin-total {
+  color: #ffffff;
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.